phoenix-core/src/main/java/org/apache/phoenix/schema/TableProperty.java

```diff
@@ -222,6 +224,28 @@ public Object getValue(Object value) {
         public Object getPTableValue(PTable table) {
             return table.useStatsForParallelization();
         }
+    },
+
+    VIEW_TTL(PhoenixDatabaseMetaData.VIEW_TTL, true, true, true) {
+        @Override
+        public Object getValue(Object value) {
+            if (value instanceof String) {
+                String strValue = (String) value;
+                if ("FOREVER".equalsIgnoreCase(strValue)) {
+                    return HConstants.LATEST_TIMESTAMP;
+                } else if ("NONE".equalsIgnoreCase(strValue)) {
```

Comment:
@jpisaac What about passing in zero as viewTTL? What would that mean? Would it have any meaning at all or some special meaning?